index.html

```diff
@@ -53,15 +53,15 @@
 
     <!-- Title -->
 
-    <title>CALCULATOR | YOSHITHA</title>
+    <title>CALCULATOR</title>
```

Comment:
This part is not assigned to you therefore undo it @Pragya2056

Reply:
This was done.